crates/astria-core/src/bridge.rs

```diff
@@ -12,3 +14,18 @@ pub struct Ics20WithdrawalFromRollupMemo {
     pub block_number: u64,
     pub transaction_hash: [u8; 32],
```

Comment:
This makes for a very ugly json. Use base64 for the hash?

Reply:
TX hashes should probably be in hex, since that is how they are looked up in cometbft